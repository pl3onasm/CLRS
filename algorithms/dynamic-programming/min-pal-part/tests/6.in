100010110111100
