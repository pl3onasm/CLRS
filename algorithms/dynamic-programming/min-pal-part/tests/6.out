Min cuts: 3
An optimal partitioning:
  10001 | 0110 | 1111 | 00
